pkg/apis/configuration/v1/kongingress_types.go

```diff
@@ -51,52 +57,97 @@ type KongIngressList struct {
 // KongIngressService contains KongIngress service configuration
 //+ It contains the subset of go-kong.kong.Service fields supported by kongstate.Service.overrideByKongIngress
 type KongIngressService struct {
+	// The protocol used to communicate with the upstream.(default:"http")
```

Comment:
You know, I was thinking about this and I realized I don't think we actually provide any default here. There's no `// kubebuilder:default` or any such thing, the protocol has to be defined by the caller right? Should we remove this?
```suggestion
	// The protocol used to communicate with the upstream.
```